main.css

```diff
@@ -301,6 +301,12 @@ body {
 	height: var(--canvas-width);
 	margin-right: var(--palette-margin-right);
 	float: left;
+	background: 
+		linear-gradient(45deg, #c0c0c0 25%, transparent 25%, transparent 75%, #c0c0c0 75%, #c0c0c0),
```

Comment:
A couple notes here:

1. Can we go with a smaller checkerboard pattern (i.e. your 2nd picture of the two that you posted in the PR)
2. While I like the two shades of grey you picked, they are very similar to the grey colors in the NES palette. I am a little worried they are too similar and could cause confusion if someone's first

I've drawn arrows pointing NES colors to the colors in the grid they look most similar to.
<img width="640" alt="Screen Shot 2024-05-29 at 12 27 13 PM" src="https://github.com/Kully/pixel-paint/assets/10369095/95fda20e-512f-44fc-b2a4-3b1ec033378b">

What do you think of this? Can we find alternative grey colors that keep the same checkerboard feel, but are visually distinguishable?

Reply:
> And maybe that review screen displays a white background or any color instead of the current #c0c0c0 and #808080 colors of that checkered grid background to show what we are drawing on it.
> You can choose the background color in that small review screen to see your current strokes more clearly.
> This will avoid confusion when colors overlap with the background.

This is a really cool idea. Reminds me of some mini-map you'd find in computer games (thinking age of empires).

For this PR and just to get this through the door, I would be okay with exploring this feature in another Pull Request. I'd want to flesh out a little more as well. :)